src/trait_manager/view/TraitsView.js

```diff
@@ -15,7 +15,6 @@ export default DomainViews.extend({
     this.className = this.pfx + 'traits';
     const toListen = 'component:toggled';
     this.listenTo(this.em, toListen, this.updatedCollection);
-    this.updatedCollection();
```

Comment:
Relates to [this issue](https://github.com/artf/grapesjs/issues/3735#issuecomment-913921482)